### PR TITLE
Compiled upload download yaml for testing purpose

### PR DESCRIPTION
**Pull request by diegolovison**
Please keep two commits.

The compiled yaml file is for manual testing. There is no Polarion Test yet.

### CODE REVIEW

**Review by jgarciao — APPROVED** [review]

**Comment by bdattoma** [comment]
lgtm, pls check the comments by Andrej

File: ods_ci/tests/Resources/Files/pipeline-samples/upload_download-compiled.yaml [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+                print("done")
+
+            import argparse
+            _parser = argparse.ArgumentParser(prog='Send file', description='')
+            _parser.add_argument("--file-size-bytes", dest="file_size_bytes", type=int, required=True, default=argparse.SUPPRESS)
+            _parser.add_argument("--outgoingfile", dest="outgoingfile", type=_make_parent_dirs_and_return_path, required=True, default=argparse.SUPPRESS)
+            _parsed_args = vars(_parser.parse_args())
+
+            _outputs = send_file(**_parsed_args)
+          image: registry.access.redhat.com/ubi8/python-38
```
[the remark this answers is not in the document]
Comment (diegolovison) [reply ods_ci/tests/Resources/Files/pipeline-samples/upload_download-compiled.yaml:85]: This is the compiled file. I can update it on next PR

File: ods_ci/tests/Resources/Files/pipeline-samples/upload_download-compiled.yaml [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+          args:
+          - --file-size-bytes
+          - '20971520'
+          - --outgoingfile
+          - $(workspaces.send-file.path)/artifacts/$ORIG_PR_NAME/$(context.taskRun.name)/outgoingfile
+          command:
+          - sh
+          - -ec
+          - |
+            program_path=$(mktemp)
```
[the remark this answers is not in the document]
Comment (diegolovison) [reply ods_ci/tests/Resources/Files/pipeline-samples/upload_download-compiled.yaml:48]: It is the compiled file

File: ods_ci/tests/Resources/Files/pipeline-samples/upload_download-compiled.yaml [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+            _parsed_args = vars(_parser.parse_args())
+
+            _outputs = send_file(**_parsed_args)
+          image: registry.access.redhat.com/ubi8/python-38
+          env:
+          - name: ORIG_PR_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.labels['custom.tekton.dev/originalPipelineRun']
+        - image: registry.redhat.io/ubi8/python-39@sha256:3523b184212e1f2243e76d8094ab52b01ea3015471471290d011625e1763af61
```
[the remark this answers is not in the document]
Comment (diegolovison) [reply ods_ci/tests/Resources/Files/pipeline-samples/upload_download-compiled.yaml:91]: Ok. Going to sync all in other PR

**Pull request merged**